src/arrays/MyArrayList.java

```diff
@@ -0,0 +1,94 @@
+package arrays;
+
+import java.util.Arrays;
+
+public class MyArrayList implements MyList {
+    private Object[] myStore;
+    private int actSize = 0;
+
+    public MyArrayList() {
+        myStore = new Object[10];
+    }
+
+    public Object get(int index) {
+        if (index < actSize) {
+            return myStore[index];
+        } else {
+            throw new ArrayIndexOutOfBoundsException();
+        }
+    }
+
+    public void add(Object obj) {
+        if (myStore.length - actSize <= 5) {
+            increaseListSize();
+        }
+        myStore[actSize++] = obj;
+    }
+
+    public Object remove(int index) {
+        if (index < actSize) {
+            Object obj = myStore[index];
+            myStore[index] = null;
+            int tmp = index;
+            while (tmp < actSize) {
+                myStore[tmp] = myStore[tmp + 1];
+                myStore[tmp + 1] = null;
+                tmp++;
+            }
+            actSize--;
+            return obj;
+        } else {
+            throw new ArrayIndexOutOfBoundsException();
+        }
+    }
+
+    public Object clear() {
+        int index = 0;
```

Comment:
these are redundant code
you can write just `for (int i = 0; i < size; i++)
            myStore[i] = null;

        size = 0;`
or `myStore = new Object[DEFAULT_CAPACITY];
size = 0`